### ISSUE TITLE
Talkback and headsets

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#393]
```
What steps will reproduce the problem?
1. connect a wired headset to your phone
2. Have someone call you on the phone, skype, hangouts
3. Talkback does not read the caller ID

What is the expected output? What do you see instead?
Caller ID is suppose to be read by talkback, but is not. It reads the caller ID 
when a wired headset is not connected to the phone

What version of the product are you using? On what operating system? 3.52 Beta 
of Talkback, but this was happening before I installed the beta. I have Android 
4.4.2


Please provide any additional information below.
Samsung Galaxy S5 Active is my phone

```

Original issue reported on code.google.com by `AriesThe...@gmail.com` on 16 Sep 2014 at 5:59
